crates/codegen/runtime/npm/package/wasm/generated/interfaces/nomic-foundation-slang-parser.d.ts

```diff
@@ -15,21 +15,60 @@ import type { TextRange } from "./nomic-foundation-slang-cst.js";
 export { TextRange };
 
 export class ParseError {
```

Comment:
I don't see the type-level documentation for many classes, like `ParseError` and `ParseOutput`. Am I missing something?

Reply:
Sounds good. We can do that in #1171